pkgs/os-specific/linux/iwd/default.nix

```diff
@@ -65,6 +67,6 @@ stdenv.mkDerivation rec {
     description = "Wireless daemon for Linux";
     license = licenses.lgpl21;
     platforms = platforms.linux;
-    maintainers = [ maintainers.mic92 ];
+    maintainers = with maintainers; [ mic92 dtzWill ];
```

Comment:
To be honest, I stopped using iwd because it does not work with my eduroam network.
You could drop me therefore as well.